web/src/recent_topics_ui.js

```diff
@@ -206,8 +206,10 @@ function set_table_focus(row, col, using_keyboard) {
             display_reply_to: $topic_row.find(".recent_topic_name a").text(),
         };
     } else {
+        const stream_name = $topic_row.find(".recent_topic_stream a").text();
+        const stream = stream_data.get_sub_by_name(stream_name);
         message = {
-            stream: $topic_row.find(".recent_topic_stream a").text(),
+            stream_id: stream?.stream_id,
```

Comment:
There is probably a `data-stream-id`  field on the row that could be accessed much more simply here.

Reply:
<img width="541" alt="image" src="https://github.com/zulip/zulip/assets/5634097/9329e850-b266-46f0-a1a6-f48b535efa0c">

I don't see one

Reply:
It looks like we've got like 3 in `recent_topic_row.hbs` in lower-down elements; and then of course there's the ID `recent_conversation:{{conversation_key}}` that encodes the stream ID (with a `:` and then the topic).

Seems like a good thing to leave for future refactoring I guess.